modules/scalar_transport/test/tests/ncp-lms/README.md

```diff
@@ -0,0 +1,200 @@
+# Summary of constraint enforcement techniques
```

Comment:
this file should be used in some capacity. looks like it's just there? is it linked in the docs?

Reply:
It's a detailed description of the tests in the form of a markdown, not in official docs. this is fine. it can be used as a starting point for detailed doco in the future.